backend/cybnetics/endpoints.py

```diff
@@ -1,8 +1,118 @@
-from flask import Flask, jsonify
+import json
+import ast
+import jwt
+import datetime
+
+from flask import Flask, jsonify, request, session
 
 app = Flask(__name__)
 app.config.from_pyfile('./config.py')
 
-@app.route('/test', methods=['GET'])
-def test():
-    return jsonify({'hello': 'world'})
+
+db = app.config['DATABASE']
+
+# This is just a sanity check from myself
+@app.route("/")
+def get_initial_response():
+    """Welcome message for the API."""
+    # Message to the user
+    message = {
+        'apiVersion': 'v1.0',
+        'status': '200',
+        'message': 'Welcome to the Flask API'
+    }
+    # Making the message looks good
+    resp = jsonify(message)
+    # Returning the object
+    return resp
+
+
+@app.route("/login", methods=['POST'])
+def login():
+    """
+    Login the user
+    """
+
+    users = db.users
+    auth
+    login_user = users.find_one({'username': request.json['username']})
+
+    try:
+
+        if login_user:
+            if(login_user['password'].encode('utf-8') == request.json['password'].encode('utf-8')):
+                exp = datetime.datetime.utcnow() + datetime.timedelta(hours=24)
+                token = jwt.encode({'username': request.json['username'], 'exp': exp}, app.config['SECRET_KEY'])
```

Comment:
like we talked about a lot of this application logic and calling the database we can move into the resources folder/function